### ISSUE TITLE
Anti-extractable nuclear antigens positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6582]
**Preferred term label:** Anti-extractable nuclear antigens positivity 

**Synonyms**
Anti-ENA

**Definition (free text, please give PubMed ID)**
A group of autoantibodies against autoantigens that can be extracted from the cell nucleus with saline. The ENAs consist of ribonucleoproteins and non-histone proteins, named by either the name of the donor who provided the prototype serum (Sm, Ro, La, Jo), or the name of the disease setting in which the antibodies were found (SS-A, SS-B, Scl-70). PMID 11899137

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity HP:0003493

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by eswietlik** [comment]
@pnrobinson 

This is a suggested structure for antinuclear antibody positivity, in the brackets, I put the action needed to achive this structure.

1. Antinuclear antibody positivity (HP:0003493)
​1.1 Extractable nuclear antigens (added)
1.1.1 Anti-Ro/SS-A and anti-La/SS-B (to be created/amended and moved as per comment on GitHub)
1.1.2 Anti-Sm (should be moved) 
1.1.3 Anti-nRNP/anti-U1-RNP (should be moved)
1.1.4 Anti-Scl-70/anti-topoisomerase I (should be moved)
1.1.5 Anti-Jo-1 (should be moved)
1.2 Anti-dsDNA (should be renamed and moved)
1.3 Anti-histone antibodies (added)
1.4 Anti-gp210 and anti-p62 (added)
1.5 Anti-centromere antibodies (should be moved)
1.6 Anti-sp100 (added)
1.7 Anti-PM-Scl  (added)
1.8 Anti-Mi2 (added)

**Comment by pnrobinson** [comment]
HP_0033476 -- Extractable nuclear antigens (previously added)

**Comment by pnrobinson** [comment]
@eswietlik  -- I think that anto-gp210 is not an ANA -- I am going to put it in the general category. If it is an ANA, please make a new issue!

**Comment by pnrobinson** [comment]
done
